src/prompts/index.tsx

```diff
@@ -58,10 +59,16 @@ export class PromptManager {
 
         this.pushContainer = rootFrame.createContainer('push');
         this.context = ctx;
+        this.pushManager = pushManager;
 
         this.setState('loading');
     }
 
+    public async hideShownPrompts(){
```

Comment:
Are the sources being properly formatted by Prettier on save?

Reply:
Formatted all of the files